README.md

```diff
@@ -141,14 +141,14 @@ wrapGifts(gifts);
 We started our counter, `i`, at `0` because arrays have zero-based indexes. Our
 condition states that we should run the code in the loop body while `i` is less
 than `gifts.length` (`3` in the above example). Our iteration, `i++`, increments
-our counter by `1` at the end of each pass through the loop. In our loop body,
-notice that we reference `gifts[i]`. Since `i` starts out as `0`, during the
-first pass through the loop `gifts[i]` is `gifts[0]` is `'teddy bear'`. During
-the second pass through the loop, `gifts[i]` is `gifts[1]` is `'drone'`. And
-during the final pass through the loop, `gifts[i]` is `gifts[2]` is `'doll'`.
-After the third pass through the loop, we increment `i` to `3`, which is no
-longer less than `gifts.length`. Our condition evaluates to `false`, and we exit
-the loop.
+our counter by `1` at the end of each pass through the loop.
+
+In our loop body, notice that we reference `gifts[i]`. Since `i` starts out as
+`0`, during the first pass through the loop `gifts[i]` is `gifts[0]` is `'teddy bear'`. During the second pass through the loop, `gifts[i]` is `gifts[1]` is
+`'drone'`. And during the final pass through the loop, `gifts[i]` is `gifts[2]`,
+which is `'doll'`. After the third pass through the loop, we increment `i` to
+`3`, which is no longer less than `gifts.length`. Our condition evaluates to
+`false`, and we exit the loop.
```

Comment:
Look at the Assignment section that follows.

I'm not sure what the student action is supposed to be here. We re-intro this "flatbook" concept (maybe it existed elsewhere?). We also provide a call to `printBadges`, but we don't define that method (do they have to write it in the tests?). This feels out of place here and we need to communicate what our expectation is or probably cut?

Reply:
rewritten to extend the existing narrative